### PR TITLE
_image.cpp formatting

### PR DESCRIPTION
**Pull request by subhayuroy**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/next_api_changes/* if API changed in a backward-incompatible way

src/_image.cpp had some documentation format problem. Fixed it and shortened over-all length of code by implementing some basic initialisation

### CODE REVIEW

**Comment by subhayuroy** [comment]
src/_image.cpp had some documentation format problems. Fixed that along with shortening over-all length of code.

**Comment by subhayuroy** [comment]
src/_image.cpp had some documentation format problems. Fixed that along with shortening over-all length of code.

File: src/_image.cpp [hunk 1]
```diff
@@ -29,16 +29,9 @@ void _bin_indices_middle(
...
-                                int nrows,
-                                const float *y,
-                                unsigned long ny,
-                                float dy,
-                                float y_min)
+void _bin_indices_middle_linear(float *arows, unsigned int *irows, int nrows, const float *y, unsigned long ny, float dy, float y_min)
 {
-    int i;
-    int ii = 0;
+    int i, ii=0;
```
Comment (timhoffm) [inline src/_image.cpp:34]: IMHO this reduces readability.

**Review by timhoffm — COMMENTED** [review]
Thanks for your interest in contributing to Matplotlib. I don't think we have a coding style guide for C++ parts of the library. So formatting is a bit of a taste thing here. For historic reasons, different files use different conventions.

The variable declaration should stay separat for readability. I'm indifferent to the other two changes.

**Comment by tacaswell** [comment]
Thank you for your work but being understanding that we did not take it.  Hopefully we will hear from you again!

**Comment by QuLogic** [comment]
> I don't think we have a coding style guide for C++ parts of the library.

We actually specify PEP7: https://matplotlib.org/devel/contributing.html#c-c-extensions, but this is something that likely came about after most of the code was written.

**Comment by QuLogic** [comment]
And much as I dislike it, PEP7 _does_ specify that style of wrapping braces for `else`.

**Pull request closed without merging**